### PR TITLE
swift syntax sugar :)

### PR DESCRIPTION
**Pull request by kostiakoval**

### CODE REVIEW

File: Source/TutorialController.swift [hunk 1]
```diff
@@ -2,9 +2,9 @@ import UIKit
 import Pages
 import Hex
 
-let MinimumMarginLateralSpace: CGFloat = 20.0
+private let MinimumMarginLateralSpace: CGFloat = 20.0
 
-@objc public class TutorialController: PagesController {
```
Comment (3lvis) [inline Source/TutorialController.swift]: Not needed for usage within Objective-C?

@objc

File: Source/TutorialController.swift [hunk 2]
```diff
@@ -80,8 +81,8 @@ public extension UIViewController {
       var view = object as! UIView
       var frame = bounds
 
-      if object.isKindOfClass(UIImageView.classForCoder()) {
-        let imageSize = (object as! UIImageView).image!.size
+      if let object = object as? UIImageView {
```
Comment (3lvis) [inline Source/TutorialController.swift]: Shiny

File: Source/TutorialModel.swift [hunk 3]
```diff
@@ -8,8 +8,8 @@ import UIKit
     get {
       return imageView.image
     }
-    set(newImage) {
-      imageView.image = newImage
+    set {
+      imageView.image = newValue
```
Comment (3lvis) [inline Source/TutorialModel.swift]: :+1:

File: Source/TutorialController.swift (hunk 1 above)
Comment (zenangst) [inline Source/TutorialController.swift]: I think we need @objc :scream:

Comment (kostiakoval) [inline Source/TutorialController.swift]: If it's inherited from NSObject you don't need it.

Comment (3lvis) [inline Source/TutorialController.swift]: Cool!

**Pull request merged**